docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -28,12 +28,15 @@ The :program:`admin/run-acceptance-tests` script has several options:
 
    Specifies what dataset backend to use for the cluster.
 
+.. option:: --branch <branch>
+
+   Specifies the branch repository from which to install packages.
+   If this is not specified, packages will be installed from the release repository.
```

Comment:
:question: Does this mean that packages will come from the latest stable release, or the latest release stable or unstable?